static/url.ts

```diff
@@ -89,20 +89,27 @@ export function deserialiseState(stateText: string): any {
     try {
         state = unrisonify(stateText);
         if (state && state.z) {
-            state = unrisonify(lzstring.decompressFromBase64(state.z));
+            const data = lzstring.decompressFromBase64(state.z);
+            // If lzstring fails to decompress this it'll return an empty string rather than throwing an error
+            if(data === '') {
```

Comment:
can you format please? And...I will FIX THE PRECOMMIT ARGH MY FAULT! :)

Reply:
Ah crap my bad, thank you for catching that!